### ISSUE TITLE
How can I still operate on the Frame?

### ISSUE THREAD

**Issue by frodriguez-hu** [nonam4/react-native-vision-camera-face-detector#28]
The plugin looks awesome btw.

**Describe the bug**
Hey, I am trying to use this plugin, and after I receive the data and wehn I want to access the original frame, I am able to do it, but the issue is that I am not able to execute any other frame processor plugin cause the Frame was marked as Closed. Any idea on how to solve this?

I am getting this error when I am trying to resize it:

```
Error: Exception in HostFunction: com.mrousavy.camera.core.FrameInvalidError: [capture/frame-invalid] Trying to access an already closed Frame! Are you trying to access the Image data outside of a Frame Processor's lifetime?
```

**To Reproduce**

This is the frame processor: 

```
    detectFaces(frame, handleFacesDetection, {
      classificationMode: 'all',
      landmarkMode: 'all',
      performanceMode: 'accurate',
      returnOriginal: true,
    });
```

And the callback: 

```
if (Object.keys(result.faces).length <= 0 || !result.frame.original) {
        return;
      }
      console.log('asd1');
      const {bounds} = result.faces[0];
      const {width, height, x, y} = bounds;
      currentBounds.current = bounds;
      try {
        const resized = resize(result.frame.original, {
          scale: {
            width: 112,
            height: 112,
          },
          crop: {
            height,
            width,
            x,
            y,
          },
          rotation: '0deg',
          pixelFormat: 'rgb',
          dataType: 'uint8',
        });
        console.log(resized);
      } catch (e) {
        console.log(e);
      }
```

**Expected behavior**

**Logs and Screenshots**
If applicable, add logs and/or screenshots to help explain your problem.

**Smartphone (please complete the following information):**
 - Vision Camera version [e.g. 1.0.0]
 - Face detector version [e.g. 1.0.0]

**Additional context**

**Comment by felire** [comment]
I resolved this by disabling the originalFrame prop, and returning the result on the detectFaces instead of executing the callback. By doing that I was able to use rhe result data on the frame for other operations, this worked fine on android I haven't tested it on ios. 
In my opinion I thini returning the result instead of the callback would be a better spproach for the plugin, what is the advantage that you have by using the callback?
